### ISSUE TITLE
How to save the changes that are being done in the layout

### ISSUE THREAD

**Issue by vinutaappzui** [katoid/angular-grid-layout#44]
Hi,Every time when user change the layout is there any way i can save this  and show them the changes he made ,when he visit the site next time

**Comment by llorenspujol** [comment]
Yes, you need to listen to the output `(layoutUpdated)="yourFunctiontoSaveLayout"` and save the layout somewhere (server, local storage...).
After that, when the user returns, you need to get the saved layout and pass it to the angular-grid-layout with the input `[layout]="mySavedLayout"`

There is not much more to discuss here, I am closing the issue. I you need more help, you can reach it on the 'discussions' section.
